source/class/qx/theme/indigo/ColorDark.js

```diff
@@ -0,0 +1,120 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2011 1&1 Internet AG, Germany, http://www.1und1.de
```

Comment:
and here

Reply:
Copyright removed per Henner's suggestion.